bitnami/mastodon/templates/init-job/init-job.yaml

```diff
@@ -0,0 +1,192 @@
+      {{- end }}
+    spec:
+      {{- include "mastodon.imagePullSecrets" . | nindent 6 }}
+      restartPolicy: OnFailure
+      {{- if .Values.initJob.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.initJob.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.initJob.hostAliases }}
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.initJob.hostAliases "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        # We separate the job in multiple containers to be able to run them in parallel. We put everything on the same job
+        # as it follows the Job Pattern best practices
+        # https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-patterns
+        {{- if or .Values.initJob.migrateDB .Values.initJob.createAdmin .Values.initJob.migrateElasticsearch }}
+        - name: migrate-and-create-admin
+          image: {{ template "mastodon.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - /bin/bash
+            - -ec
+          args:
+            - /scripts/migrate-and-create-admin.sh
+          {{- if .Values.initJob.containerSecurityContext.enabled }}
+          securityContext: {{- omit .Values.initJob.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" (or .Values.image.debug .Values.diagnosticMode.enabled) | quote }}
+            - name: MASTODON_DATABASE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "mastodon.database.secretName" . }}
+                  key: {{ include "mastodon.database.passwordKey" . | quote }}
+            {{- if (include "mastodon.redis.auth.enabled" .) }}
+            # The rake db:migrate job requires access to Redis
+            - name: MASTODON_REDIS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "mastodon.redis.secretName" . }}
+                  key: {{ include "mastodon.redis.passwordKey" . | quote }}
+            {{- end }}
+            {{- if (include "mastodon.elasticsearch.auth.enabled" .) }}
+            - name: MASTODON_ELASTICSEARCH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "mastodon.elasticsearch.secretName" . }}
+                  key: {{ include "mastodon.elasticsearch.passwordKey" . | quote }}
+            {{- end }}
+            {{- if .Values.initJob.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.initJob.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "mastodon.defaultConfigmapName" . }}
+            - configMapRef:
```

Comment:
I think this the reason why this configmap is created empty...

Reply:
Fixed